provision-bastion.sh

```diff
@@ -12,7 +12,7 @@ echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https:/
 apt-get update
 apt-get upgrade -y
 
-apt-get -y install apt-transport-https azure-cli ca-certificates curl gnupg jq kubectl lsb-release nmap openjdk-11-jre-headless postgresql tcpdump parallel redis-server
+apt-get -y install apt-transport-https azure-cli ca-certificates curl gnupg jq kubectl lsb-release nmap openjdk-17-jre-headless postgresql tcpdump parallel redis-server
```

Comment:
is anyone using openjdk-11? removing this could cause issues for them

Reply:
probably safer to have both if you can for now

Reply:
Good idea

Reply:
I have updated the PR to include both openjdk-11-jre-headless & openjdk-17-jre-headless